### DISCUSSION TITLE
Are all borders in the new GPUI2 interface permanent?

### DISCUSSION THREAD

**Discussion by Tom-Boscher** [zed-industries/zed#6664] [Feature Requests]
I notice that there is now a lot of borders around elements in the new interface using GPUI2. I personally preferred the old interface with less borders as it was more minimalist.
Does this new design is temporary or will it be the new design interface for Zed?

<img width="1512" alt="Capture d’écran 2024-01-14 à 23 05 20" src="https://github.com/zed-industries/community/assets/24520681/dc794f0a-3baa-449e-a2c9-86933b380783">
<img width="1512" alt="Capture d’écran 2024-01-14 à 23 05 27" src="https://github.com/zed-industries/community/assets/24520681/cb826b7c-03a9-49f0-9db2-9f37cb858e11">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
